apache-rat-tools/src/main/java/org/apache/rat/tools/AntDocumentation.java

```diff
@@ -55,8 +47,6 @@
  * A simple tool to convert CLI options into an Ant report base class.
  */
 public final class AntDocumentation {
```

Comment:
@Claudenw class contains 3 unused methods - are these called by reflection as well and we need some more tests?